backend/src/api/post-api.js

```diff
@@ -70,21 +70,39 @@ router.post('/login', async (req, res) => {
     signInWithEmailAndPassword(auth, email, password).then((userCredential) => {
         const user = userCredential.user;
         console.log("login email:", user.email);
+        const membership = Membership.findOne({ accountId: user.uid });
+        if(membership === null) {
+            return res.status(200).send({
+                status: 'success',
+                message: 'User logged in',
+                data: {
+                    accountId: user.uid,
+                    email: user.email,
+                    isOrg: false,
```

Comment:
@Donnyd88 , I added `isOrg` variable to help you to determine if the user is an organization or not. So that you can redirect the user to the `userpage` if `isOrg` is false. and to `ord-dashboard` if `isOrg` is true.

Reply:
Added